controllers/users.js

```diff
@@ -3,39 +3,41 @@ const util = require('../util')
 const collectionsApi = require('./collections')
 const userPermissions = require('../middleware/users_permissions')
 
-exports.login = async (req) => {
+exports.login = async (req, collection) => {
   const password = req.body.password
 
   if (typeof req.body.email !== 'string') {
     throw new util.ApiError(400, 'email must be a string')
   }
 
   // check if user exists
-  const result = await req.db.users.find({
+  const result = await req.db[collection].find({
     email: req.body.email
   })
   if (result.length === 0) {
-    throw new util.ApiError(400, 'No user found with this email.')
+    throw new util.ApiError(400, 'No ' + collection + ' found with this email.')
   }
   const user = result[0]
   if (!auth.isValidPassword(password, user.password)) {
     throw new util.ApiError(401, 'Incorrect password')
   }
-  const payload = auth.doLogin(user, req)
+  const jwt_options = req.settings.jwt_expires_in ? { expiresIn: req.settings.jwt_expires_in } : {}
```

Comment:
@thomas4019 something i snuck into this PR, to keep behaviour the same i defaulted to nothing for `jwt_expires_in` so they still never expire - but wondering if i should default it to something large like `1y`? Would also neaten up this code a bit as wouldn't have to do so many checks

Reply:
leaving as is so not to interfere with existing applications, but still think its a good idea for future